src/calendar/tests/CalendarSelection.spec.js

```diff
@@ -5,23 +5,13 @@ import CalendarSelection from '../CalendarSelection';
 
 describe('The CalendarSelection Component', function () {
   beforeEach(function () {
-    this.spyCx = spyOn(CalendarSelection.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
-
     var shallowRenderer = TestUtils.createRenderer();
-    shallowRenderer.render(<CalendarSelection pending={true} modifier='test'/>);
+    shallowRenderer.render(<CalendarSelection pending={true} modifier='test' bemBlock='DateRangePicker' />);
     this.renderedComponent = shallowRenderer.getRenderOutput();
   });
 
   it('should render the right element', function () {
     expect(this.renderedComponent.type).toBe('div');
-    expect(this.spyCx).toHaveBeenCalledWith({
-      states: {
-        pending: true,
-      },
-      modifiers: {
-        test: true,
-      },
-    });
-    expect(this.renderedComponent.props.className).toEqual('my-class');
+    expect(this.renderedComponent.props.className).toContain('CalendarSelection');
```

Comment:
Given that I set the bemBlock attribute to DateRangePicker, it would make sense to test against an actual class name rather than a partial name indeed. I will change it.